Source/WebKit/Platform/cocoa/CocoaHelpers.h

```diff
@@ -78,4 +83,8 @@ NSString *escapeCharactersInString(NSString *, NSString *charactersToEscape);
 NSDate *toAPI(const WallTime&);
 WallTime toImpl(NSDate *);
 
+NSSet *toAPI(HashSet<String>&);
+NSSet *toAPI(MatchPatternSet&);
```

Comment:
This should move to `WebExtensionMatchPattern.h` and `WebExtensionMatchPattern.cpp` outside the `class { }`